src/AppRoot.tsx

```diff
@@ -40,72 +39,19 @@ const AppRoot: React.FC = () => {
   const dispatch = useAppDispatch();
   const location = useLocation();
   const navigate = useNavigate();
-
+  const telemetry = useTelemetry();
   const apiEndpoint = useApiEndpoint();
 
-  const {data: clusterConfig, refetch: refetchClusterConfig} = useGetClusterConfigQuery();
-
-  const [isCookiesVisible, setCookiesVisibility] = useState(!localStorage.getItem('isGADisabled'));
-  const [featureFlags, setFeatureFlags] = useState<string[]>([]);
```

Comment:
are we going to be able to use feature flags from posthog also in the future?

Reply:
We could add that feature later, it shouldn't be hard; we may inject them from GTM by a callback 👍 I want to reduce complexity for the initial GTM release, as it's already a big change for us

Reply:
Added a follow-up ticket for getting it back: https://github.com/kubeshop/testkube/issues/4187